src/lib/graphs.js

```diff
@@ -1,5 +1,6 @@
-"use strict";
+'use strict';
```

Comment:
can you please make sure prettier is set to format on save?
for instructions see https://trello.com/c/lVydBxrE/12-coding-style